rules/windows/execution_via_hidden_shell_conhost.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/08/17"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects suspicious console activity which can be a a result of code injection. Verify parent process execution details
+such as network connections and file writes for context verification.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Console Process"
+references = ["https://www.fireeye.com/blog/threat-research/2017/08/monitoring-windows-console-activity-part-one.html"]
+risk_score = 73
+rule_id = "05b358de-aa6d-4f6c-89e6-78f74018b43b"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type:(start or process_started) and
```

for process events, I would be explicit be using "start" because there is also an event.type of info which may result in unrelated or duplicative alerts. process_started is for backwards compat for pre 7.8 beats